src/config.cpp

```diff
@@ -1,29 +1,37 @@
 #include "config.h"
 #include "signals.h"
+#include "md5.h"
 
 using openxc::pipeline::Pipeline;
 using openxc::interface::uart::UartDevice;
 using openxc::payload::PayloadFormat;
 
 namespace usb = openxc::interface::usb;
+namespace telit = openxc::telitHE910;
 
 namespace signals = openxc::signals;
 
+static void getFlashHash(openxc::config::Configuration* config);
+
 static void initialize(openxc::config::Configuration* config) {
     config->pipeline = {
         &config->usb,
         &config->uart,
+		&config->telit,
 #ifdef __USE_NETWORK__
         &config->network,
 #endif // __USE_NETWORK__
     };
+	// run flashHash
+	getFlashHash(config);
+	config->telit.uart = &config->uart;
     config->initialized = true;
 }
 
 openxc::config::Configuration* openxc::config::getConfiguration() {
     static openxc::config::Configuration CONFIG = {
         messageSetIndex: 0,
-        version: "7.0.1-dev",
+        version: "7.0.1-c5-dev",
```

Comment:
You probably didn't mean to leave this in here - we could make it more dynamic, so the version is set based on the platform.